src/multi_select.coffee

```diff
@@ -22,6 +22,11 @@ Ember.Widgets.MultiSelectComponent = Ember.Widgets.SelectComponent.extend
   placeholder: undefined
   persistentPlaceholder: undefined
 
+  # disable tabindex of the component container to set focus directly to
+  # the input field, which is always visible. This helps reducing one tab
+  # step to navigate back to the previous component
+  tabindex: -1
```

Comment:
@azirbel I already had the tabindex here overridden as -1 as you mentioned in the other comment